data/json/monsters.json

```diff
@@ -114,6 +114,140 @@
     "categories":["WILDLIFE"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_beakhound",
+    "name": "beakhound",
+    "species":"MAMMAL",
+    "default_faction":"dog",
+    "symbol":"d",
+    "color":"green",
+    "size":"SMALL",
+    "material":"flesh",
+    "diff":5,
+    "aggression":2,
+    "morale":15,
+    "speed":150,
+    "melee_skill":6,
+    "melee_dice":2,
+    "melee_dice_sides":7,
+    "melee_cut":5,
+    "dodge":3,
+    "armor_bash":0,
+    "armor_cut":0,
+    "vision_day":40,
+    "vision_night":5,
+    "hp":30,
+    "special_attacks":[["INK_JET", 10]],
+    "death_function":"NORMAL",
+    "description": "A strange dog which has a sharp beak instead of it's usual canine snout. It's large bulging eyes give it a silly appearance.",
+    "flags":["SEES", "HEARS", "SMELLS", "ANIMAL", "WARM", "FUR", "HIT_AND_RUN", "KEENNOSE", "BLEED", "ATTACKMON", "BONES", "FAT", "INK_SAC", "MUT_ABLE"],
+    "anger_triggers":["HURT", "FRIEND_ATTACKED", "FRIEND_DIED"],
```

Comment:
Not really important (all beakdogs will only ever be spawned friendly), but it would be good if the dogs also had "PLAYER_WEAK" or "PLAYER_CLOSE" anger trigger, because currently they have no way of going hostile.
That way they could be spawned feral somewhere.